adafruit_epd/mcp_sram.py

```diff
@@ -80,24 +87,24 @@ def read(self, addr, length, reg=SRAM_READ):
             spi.readinto(buf)  # pylint: disable=no-member
         return buf
 
-    def read8(self, addr, reg=SRAM_READ):
+    def read8(self, addr: int, reg: int = SRAM_READ):
         """read a single byte at the passed address"""
         return self.read(addr, 1, reg)[0]
 
-    def read16(self, addr, reg=SRAM_READ):
+    def read16(self, addr: int, reg: int = SRAM_READ):
         """read 2 bytes at the passed address"""
         buf = self.read(addr, 2, reg)
         return buf[0] << 8 | buf[1]
 
-    def write8(self, addr, value, reg=SRAM_WRITE):
+    def write8(self, addr: int, value: Any, reg: int = SRAM_WRITE):
         """write a single byte at the passed address"""
         self.write(addr, [value], reg)
 
-    def write16(self, addr, value, reg=SRAM_WRITE):
+    def write16(self, addr: int, value: Any, reg: int = SRAM_WRITE):
```

Comment:
Same as the other, I think the code is expecting `value` to be `int` in this case